diff --git a/.gitignore b/.gitignore
index 11c8052..8d0c0d4 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,8 +1,8 @@
 .gradle/
 .idea/
 
-petal-api/
-petal-server/
+dogal-api/
+dogal-server/
 
 build-data/
 build/
diff --git a/build.gradle.kts b/build.gradle.kts
index 449e75a..48c2214 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -24,6 +24,8 @@ subprojects {
     apply(plugin = "java")
 
     java {
+        sourceCompatibility = JavaVersion.VERSION_16
+        targetCompatibility = JavaVersion.VERSION_16
         toolchain {
             languageVersion.set(JavaLanguageVersion.of(17))
         }
@@ -48,7 +50,7 @@ subprojects {
 }
 
 paperweight {
-    serverProject.set(project(":petal-server"))
+    serverProject.set(project(":dogal-server"))
 
     remapRepo.set("https://maven.fabricmc.net/")
     decompileRepo.set("https://files.minecraftforge.net/maven/")
@@ -64,8 +66,8 @@ paperweight {
             apiPatchDir.set(layout.projectDirectory.dir("patches/api"))
             serverPatchDir.set(layout.projectDirectory.dir("patches/server"))
 
-            apiOutputDir.set(layout.projectDirectory.dir("petal-api"))
-            serverOutputDir.set(layout.projectDirectory.dir("petal-server"))
+            apiOutputDir.set(layout.projectDirectory.dir("dogal-api"))
+            serverOutputDir.set(layout.projectDirectory.dir("dogal-server"))
         }
     }
 }
diff --git a/gradle.properties b/gradle.properties
index 6603554..fa0de6c 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -2,7 +2,7 @@ org.gradle.parallel=true
 org.gradle.caching=true
 org.gradle.vfs.watch=false
 
-group=host.bloom.petal
+group=jp.runq.dogal
 version=1.19.2-R0.1-SNAPSHOT
 mcVersion=1.19.2
 purpurRef=afd4044046331919443aa0ae9b53ae197a95ba3b
diff --git a/settings.gradle.kts b/settings.gradle.kts
index c09704d..6452dc4 100644
--- a/settings.gradle.kts
+++ b/settings.gradle.kts
@@ -5,5 +5,5 @@ pluginManagement {
     }
 }
 
-rootProject.name = "petal"
-include("petal-api", "petal-server")
+rootProject.name = "dogal"
+include("dogal-api", "dogal-server")
