From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Taitaitatata <70869837+Taitaitatata@users.noreply.github.com>
Date: Mon, 15 May 2023 01:17:32 +0900
Subject: [PATCH] =?UTF-8?q?[Update]=20petal=E3=81=8B=E3=82=89docal?=
 =?UTF-8?q?=E3=81=AB=E5=90=8D=E5=89=8D=E3=82=92=E5=A4=89=E6=9B=B4?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit


diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index 9302bd9c75011f395b7f7866f59a5a7122147051..71224872d893e867588b6d22089db3396eda02ec 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -1687,7 +1687,7 @@ public abstract class MinecraftServer extends ReentrantBlockableEventLoop<TickTa
         final String purpurModName = org.purpurmc.purpur.PurpurConfig.serverModName; // Purpur - Purpur > // Paper - Paper > // Spigot - Spigot > // CraftBukkit - cb > vanilla!
 
         return switch (purpurModName) {
-            case "Purpur" -> "petal";
+            case "Purpur" -> "docal";
             default -> purpurModName;
         };
         // petal end
diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
index a05e74a74eaa7689656e2f13bbde992af226e404..cc58097a575c2d5464f18575582f3db8a8caca83 100644
--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
@@ -249,7 +249,7 @@ import javax.annotation.Nullable; // Paper
 import javax.annotation.Nonnull; // Paper
 
 public final class CraftServer implements Server {
-    private final String serverName = "petal"; // Paper // Purpur // petal
+    private final String serverName = "docal"; // Paper // Purpur // petal // docal
     private final String serverVersion;
     private final String bukkitVersion = Versioning.getBukkitVersion();
     private final Logger logger = Logger.getLogger("Minecraft");
